CMSgov/mint-app
# [EASI-3945] Notification for Collaborator added

# EASI-3945


## Changes and Description

- BE: https://github.com/CMSgov/mint-app/pull/1003
- FE: https://github.com/CMSgov/mint-app/pull/1031



## How to test this change
High overview functionality check

1. Add new collaborator to model plan
2. Verify that new collaborator gets notification in notification center and email 


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
